core/src/com/unciv/ui/images/IconTextButton.kt

```diff
@@ -44,6 +42,11 @@ open class IconTextButton(
 
     init {
         pad(10f)
-        labelCell.padTop(10f - Fonts.getDescenderHeight(fontSize));
+        // aligned with icon
+        labelCell.padTopDescent()
     }
 }
+
+fun Cell<Label>.padTopDescent(): Cell<Label> {
+    return padTop(Fonts.getDescenderHeight(Fonts.ORIGINAL_FONT_SIZE.toInt()) * actor.fontScaleY)
```

Comment:
Not entirely sure if this is the correct calculation, but it works well for the test cases I tried:
- different Screen Sizes set in the options
- different Font Size Multiplier set in the options